server/plugin/subscriptions.go

```diff
@@ -255,6 +273,66 @@ func (p *Plugin) StoreSubscriptions(s *Subscriptions) error {
 	return nil
 }
 
+func (p *Plugin) GetExcludedNotificationRepos() ([]string, error) {
+	var subscriptions []string
+	value, appErr := p.API.KVGet(SubscribedRepoNotificationOff)
```

Comment:
Is there one global list of `SubscribedRepoNotificationOff`? What happens if there are multiple  orgs with disabled repos?

Reply:
@hanzei  he have taken care of this by storing the `org/repo` so that it won't have any issue when we have multiple orgs disabled